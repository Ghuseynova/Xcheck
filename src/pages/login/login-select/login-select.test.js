import React from 'react';
import { render } from "@testing-library/react";
import LoginSelect from './login-select';

describe('render', () => {    
  test('LoginSelect', () => {
    const roles = ['author', 'student', 'supervisor', 'course_manager'];
    const result = render(
      <LoginSelect
        roles={roles} />
    );
    expect(result).toMatchSnapshot()
  });
});