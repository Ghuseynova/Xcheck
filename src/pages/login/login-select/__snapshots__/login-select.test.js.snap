// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render LoginSelect 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="ant-select ant-select-multiple ant-select-allow-clear ant-select-show-search"
        style="width: 100%;"
      >
        <div
          class="ant-select-selector"
        >
          <span
            class="ant-select-selection-search"
            style="width: 0px;"
          >
            <input
              aria-activedescendant="rc_select_TEST_OR_SSR_list_0"
              aria-autocomplete="list"
              aria-controls="rc_select_TEST_OR_SSR_list"
              aria-haspopup="listbox"
              aria-owns="rc_select_TEST_OR_SSR_list"
              autocomplete="off"
              class="ant-select-selection-search-input"
              id="rc_select_TEST_OR_SSR"
              readonly=""
              role="combobox"
              style="opacity: 0;"
              unselectable="on"
              value=""
            />
            <span
              aria-hidden="true"
              class="ant-select-selection-search-mirror"
            >
              
               
            </span>
          </span>
          <span
            class="ant-select-selection-placeholder"
          >
            Please select role(s)
          </span>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="ant-select ant-select-multiple ant-select-allow-clear ant-select-show-search"
      style="width: 100%;"
    >
      <div
        class="ant-select-selector"
      >
        <span
          class="ant-select-selection-search"
          style="width: 0px;"
        >
          <input
            aria-activedescendant="rc_select_TEST_OR_SSR_list_0"
            aria-autocomplete="list"
            aria-controls="rc_select_TEST_OR_SSR_list"
            aria-haspopup="listbox"
            aria-owns="rc_select_TEST_OR_SSR_list"
            autocomplete="off"
            class="ant-select-selection-search-input"
            id="rc_select_TEST_OR_SSR"
            readonly=""
            role="combobox"
            style="opacity: 0;"
            unselectable="on"
            value=""
          />
          <span
            aria-hidden="true"
            class="ant-select-selection-search-mirror"
          >
            
             
          </span>
        </span>
        <span
          class="ant-select-selection-placeholder"
        >
          Please select role(s)
        </span>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
